cmd/cmd.js

```diff
@@ -340,7 +341,8 @@ class Cmd {
           logFile: options.logfile,
           skipUndeployed: options.skipUndeployed,
           skipFunctions: options.skipFunctions,
-          skipEvents: options.skipEvents
+          skipEvents: options.skipEvents,
+          output: options.output || './diagram.svg'
```

Comment:
Can we put this `diagram.svg` in a constant somewhere? :)

Sth. like `DEFAULT_DIAGRAM_PATH` or so

Reply:
All right, I will update the PR today evening!